{% extends "ipscdb/default.html" %}

{% block content %}
<div class="header-image">
  <img src="static/img/header.png"/>
</div>
<div class="tinted large">
  <p>
  <b>The iPS cells phenogenetic map project "iPhemap"</b> is a comprehensive database
  that aims to provide a field synopsis and catalog in vitro neuronal disease
  phenotypes from induced pluripotent stem cells (iPSCs), derived from patients
  with neurological diseases. You can <a href="search">search cellular and molecular phenotypes </a>
  from <a href="/studies">62 published reports</a>. We characterized 530 distinct cellular phenotypes
  and the resulting relationships between gene and phenotype into a phenogenetic
  map that can be used to build new hypotheses in the field of neurological
  disease modeling, and to identify potential new opportunities to design novel
  drug strategies. The project comprises a comprehensive catalog of phenogenetic
  profiles from patient derived-iPSCs from highly curated, published reports and
  returns: 1) Cellular phenotypes from iPSCs, Neural stem cells,
  oligodendrocytes, astrocytes, and neurons with genetic mutations linked to
  neurological diseases. 2) Molecular phenotypes and dysregulated pathways when
  available, which stem from microarray analysis, and derived from the functional
  annotations associated with previously established genome ontology. 
  </p>
  <!--
  <p>
  iPhemap is comprehensive regularly updated database that
  aims to provide a field synopsis and catalog all of the in vitro
  neuronal disease phenotypes from induced pluripotent stem
  cells (iPSCs) derived from patients with neurological diseases.
  </p>

  <p>
  You can <a href="search">search cellular and molecular phenotypes </a>
  from <a href="/studies">62 published reports</a>. We characterized 428 distinct
  cellular phenotypes and the resulting relationships between
  gene and phenotype into phenogenetic map that can be used
  to build new hypothese sin the field of neurological disease
  modeling, and to identify potential new opportunities to
  design novel drug strategies.
  </p>
  
  <p>
  iPheMap comprises a comprehensive catalog of phenogenetic profiles from patient
  derived-iPSCs from highly curated, published reports and returns:
  </p>

  <ol>
    <li>
    Cellular phenotypes reporting morphological,
    functional, and process changes from iPSCs, NSCs,
    oligodendrocytes, astrocytes, and neurons with
    genetic mutations linked to neurological diseases.
    </li>

    <li>
    Molecular phenotypes, which stem from the
    microarray analysis, are derived from the functional
    annotations associated with previously established
    genome ontology. These molecular phenotypes
    are presented as treemaps, sized according to their
    degree of statistical significance.
    </li>

    <li>
    Dysregulated pathways also arising from genome
    ontology, which reflect disrupted canonical
    pathways.
    </li>
  </ol>

  <p>For details on the background and system biology methods used in iPhemap, see the <a href="research">Methods</a> section. The website is developed and maintained by the <a href="research">Imitola Laboratory</a>.</p>
  -->
</div>

<!--
<div class="tinted">
  <img src="static/img/explain.png" style="width: 100%;">
</div>
--!>

<div class="tintedb">
  <h4>Citing iPhemap</h4>
  <p>Hollingsworth E, Vaughn JE, Orack JC, Skinner C, Racke MK, Khouri J, Watanabe F, Kosik KS, Imitola J  
  <br><b>iPhemap</b>: An atlas of phenotype to genotype relationships of human induced pluripotent stem cell models of neurological diseases. 2016 <i>Submitted</i></p>
</div>


<div class="tintedc">
  <div class="tintedb">
    <h4>Announcements</h4>

    {% for announcement in announcements %}
    <a class="announcement" href="{{ announcement.link }}">
      <span class="announcement-date">{{ announcement.date }}:</span>
      <span>{{ announcement.title }}</span>
    </a>
    {% endfor %}
  </div>
  <div class="tintedb blue-links">
    <h4>Links</h4>
    <a class="announcement" href="https://www.nimhgenetics.org">NIMH Repository and Genomics Resource</a>
    <a class="announcement" href="https://www.nimhgenetics.org/stem_cells/">The NIMH Stem Cell Center, hiPSC cell line data biorepositories</a> 
    <a class="announcement" href="http://www.ncbi.nlm.nih.gov/gap">Database of Genotypes and Phenotypes (dbGaP)</a>
    <a class="announcement" href="http://rdocdb.nimh.nih.gov/">Clinical phenotyping data through the NIMH Data Archive</a>
  </div>
</div>

<div class="tintedb">
  <h4>Development</h4>
  <p>The <a href="/research">Imitola Laboratory</a> at The Ohio State University Neurological Institute has developed this resource for identifying phenotype-genotype relationship and cataloging curated phenotypes  in patient-derived models of neurological disease.</p>
</div>

<div class="tintedb">
  <h4>Version</h4>
  <p>The current version number is 1.1 - 122015. The most recent update to data was on date</p>
</div>

{% endblock %}
