{% extends "ipscdb/default.html" %}

{% block content %}
<div class="tinted">
  <h4>About</h4>
  <p>
  The <b>iPS cell phenogenetic map project “iPhemap”</b> is comprehensive 
  database that aims to provide a field synopsis and catalog all of the 
  <i>in vitro</i> neuronal disease phenotypes from induced pluripotent stem
  cells (iPSCs) derived from patients with neurological diseases.
  You can <a href="search">search cellular and molecular phenotypes </a>
  from <a href="/studies">79 published reports</a>. We characterized 550 
  distinct  cellular phenotypes and the resulting relationships between
  gene and phenotype into a phenogenetic map that can be used to build 
  new hypotheses in the field of neurological disease modeling, and to 
  identify potential new opportunities to design novel drug strategies. 
  The project comprises a comprehensive catalog of phenogenetic profiles 
  from patient derived-iPSCs from highly curated, published reports  
  and returns: 1) Cellular phenotypes from iPSCs, neural stem cells, 
  oligodendrocytes, astrocytes, and neurons with genetic mutations linked
  linked to neurological diseases. 2) Molecular phenotypes and dysregulated 
  pathways, when available, from gene ontology analyses of gene expression 
  profiles.
  </p>

  <div class="tintedb">
  <h4>Citing iPhemap</h4>
  <p>Hollingsworth, E, Vaughn JE, Orack JC, Skinner C, Racke MK, Khouri J, Watanabe F, Kosik KS, Imitola J.  
  <br>he iPhemap Database: An atlas of phenotype to genotype relationships of Human induced pluripotent stem cell models of neurological diseases, systematic analysis, research practices, and reporting bias (2017) <i>Submitted</i></p>
</div>

<div class="tintedc">
  <div class="tintedb">
    <h4>Announcements</h4>

    {% for announcement in announcements %}
    <a class="announcement" href="{{ announcement.link }}">
      <span class="announcement-date">{{ announcement.date }}:</span>
      <span>{{ announcement.title }}</span>
    </a>
    {% endfor %}
  </div>
  <div class="tintedb blue-links">
    <h4>Links</h4>
    <a class="announcement" href="https://www.nimhgenetics.org">NIMH Repository and Genomics Resource</a>
    <a class="announcement" href="https://www.nimhgenetics.org/stem_cells/">The NIMH Stem Cell Center, hiPSC Cell Line Data biorepositories</a> 
    <a class="announcement" href="http://www.ncbi.nlm.nih.gov/gap">Database of Genotypes and Phenotypes (dbGaP)</a>
    <a class="announcement" href="http://rdocdb.nimh.nih.gov/">Clinical Phenotyping Data through the NIMH Data Archive</a>
    <a class="announcement" href="https://genome.ucsc.edu/">Genome Browser</a>
    <a class="announcement" href="http://www.genome.jp/kegg/genome.html">KEGG Genome</a>
    <a class="announcement" href="https://www.encodeproject.org/">ENCODE</a>
    <a class="announcement" href="https://www.neuroscienceblueprint.nih.gov/connectome/">NIH Connectome</a>
  </div>
</div>

<div class="tintedb">
  <h4>Development</h4>
  <p>The <a href="/research">Imitola Laboratory</a> at The Ohio State University Neurological Institute has developed this resource for identifying phenotype-genotype relationships and cataloging curated phenotypes in patient-derived models of neurological disease.</p>
</div>

<div class="tintedb">
  <h4>Version</h4>
  <p>The current version number is 1.1 - 122016. The most recent update to data was on date</p>
</div>

{% endblock %}
